src/anybody.js

```diff
@@ -1,25 +1,34 @@
+
 import Prando from 'prando'
 
 
 import EventEmitter from 'events'
-import Sound from './sound.js'
+// import Sound from './sound.js'
 import { Visuals } from './visuals.js'
-import { utils, Calculations } from './calculations.js'
-const {
+import {
+  _smolr,
+  _convertBigIntToModP,
+  _approxDist,
+  _approxSqrt,
+  _approxDiv,
+  _calculateTime,
+  _explosion,
+  _addVectors,
+  _validateSeed,
+  Calculations
+} from './calculations.js'
+
+export {
   _smolr,
-  // _convertBigIntToModP,
-  // _approxDist,
-  // _approxSqrt,
-  // _approxDiv,
-  // _calculateTime,
-  // _explosion,
-  // _addVectors,
+  _convertBigIntToModP,
+  _approxDist,
+  _approxSqrt,
+  _approxDiv,
+  _calculateTime,
+  _explosion,
+  _addVectors,
   _validateSeed
-} = utils
-
-export { utils }
-
-
+}
```

Comment:
Re-exporting here makes it so all of these will be included in the bundle if anybody is included (whether or not they are all used). Instead I'd suggest removing this export and just importing what you need where you need it directly from `calculations.js`.